webpack/assets/javascripts/react_app/common/colors.scss

```diff
@@ -9,6 +9,7 @@ $pf-green-600: #1e4f18;
 $pf-blue-400: #0088ce;
 $pf-color-white: #fff;
 $pf-border-gray: #d1d1d1;
+$pf-global--warning-color--100: #f0ab00;
```

Comment:
Looking at the bootstrap docs:
https://5c507d49471426000887a6a7--react-bootstrap.netlify.app/components/forms/

It looks like we should be able to use the `validationState` prop in react and set it to `warning` to make it orange.